infrastructure/lib/constructs/robomo-discord-bot.ts

```diff
@@ -20,12 +20,10 @@ export class RobomoDiscordBot extends Construct {
     const cluster = new ecs.Cluster(this, 'DiscordBotFargateCluster');
 
     const imageAsset = new DockerImageAsset(this, 'DiscordBotECSDockerImage', {
-      directory: path.join(__dirname, '../../robomo-discord-bot'),
+      directory: path.join(__dirname, '../../../robomo-discord-bot'),
```

Comment:
:whoa: how was this working at all if we had that path wrong? 😕

Reply:
not sure 😅